### PR TITLE
[ISV-933] Merge the operator-pipeline-images code to operator-pipeline code

### PR DESCRIPTION
**Pull request by haripate**
**The Repo structure is shown below:**

Screenshot from 2021-11-10 17-31-23

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
```
Comment (MarcinGinszt) [inline .github/workflows/main.yml:1]: Could you change the name of this file to something more descriptive? Now it's not a main workflow, it's just the image workflow :)

File: .gitignore [hunk 2]
```diff
@@ -13,3 +13,146 @@ kubeconfig
...
+
+# mypy
+.mypy_cache/
+.dmypy.json
+dmypy.json
+
+# Pyre type checker
+.pyre/
+
+# VSCode project settings
```
Comment (MarcinGinszt) [inline .gitignore:150]: Stuff below is duplicated, it already existed in the .gitignore

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+name: Build and Push Image
+
+on:  # yamllint disable-line rule:truthy
+  - push
```
Comment (MarcinGinszt) [inline .github/workflows/main.yml:5]: @Allda I know it was working this way in the `operator-pipelines-images`, but should we really build and push image on each push to repo?

File: .github/workflows/release.yml [hunk 4]
```diff
@@ -0,0 +1,50 @@
+---
```
Comment (MarcinGinszt) [inline .github/workflows/release.yml:1]: Again, could you give it more descriptive name? Eg. release-image.yml

File: .dockerignore [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# Ignoring git and cache folders
```
Comment (MarcinGinszt) [inline .dockerignore:1]: We don't need the whole pipelines code in the image- you can also ignore `templates` and `ansible`.

**Review by MarcinGinszt — CHANGES_REQUESTED** [review]
I think that all the files that are necessary only for the `operator-pipeline-images`, should be in the directory `operator-pipeline-images` I mean `certs`, `requirements.txt`, `requirements-dev.txt` and- if you agree with me- also setup.py and dockerfile. WDYT?

File: .github/workflows/main.yml (hunk 3 above)
Comment (Allda) [reply .github/workflows/main.yml:5]: Yeah, why not? This allows users to build images and test them without manually building and pushing images to the registry.

**Comment by wcheang** [comment]
Files like requirements*.txt and tox.ini should be moved to the root level and adjusted accordingly

**Review by amisstea — APPROVED** [review]

**Review by wcheang — APPROVED** [review]

**Comment by wcheang** [comment]
If the resulting image has been tested to work, then LGTM :+1:

**Pull request merged**